Classes/Updater/Yellowpages2SlugUpdater.php

```diff
@@ -101,10 +104,10 @@ public function executeUpdate(): bool
                     )
                 )
             )
-            ->execute();
+            ->executeQuery();
 
         $connection = $this->getConnectionPool()->getConnectionForTable($this->tableName);
-        while ($recordToUpdate = $statement->fetch()) {
+        while ($recordToUpdate = $statement->fetchOne()) {
```

Comment:
executeQuery returns a Result object instead of Statement. Please update $statement to $queryResult

Reply:
Done ☑️